build/PackageVersions.targets

```diff
@@ -7,6 +7,7 @@
   <ItemGroup>
     <PackageReference Update="Microsoft.AspNet.WebApi.Client" Version="5.2.6" />
     <PackageReference Update="Moq" Version="4.10.1" PrivateAssets="All" />
-    <PackageReference Update="Newtonsoft.Json" Version="11.0.2" PrivateAssets="All" />
+    <PackageReference Update="Newtonsoft.Json" Version="12.0.1" PrivateAssets="All" />
```

Comment:
Just out of curiosity, what required the upgrade in Json.Net? Swashbuckle?

Reply:
yup.